File: packages/mesh-io/typescript/src/extension-to-point-set-io.ts
const extensionToPointSetIo = new Map([
  ["vtk", "vtk"],
  ["obj", "obj"],
  ["off", "off"],
  ["mz3", "mz3"],
  ["iwm", "wasm"],
  ["iwm.cbor", "wasm"],
  ["iwm.cbor.zst", "wasm-zstd"],
]);

export default extensionToPointSetIo;

File: packages/mesh-io/python/itkwasm-mesh-io-wasi/itkwasm_mesh_io_wasi/extension_to_point_set_io.py
from collections import OrderedDict

extension_to_point_set_io = OrderedDict([
    ('.vtk', 'vtk_poly_data'),
    ('.obj', 'obj'),
    ('.off', 'off'),
    ('.iwm', 'wasm'),
    ('.iwm.cbor', 'wasm'),
    ('.iwm.cbor.zst', 'wasm_zstd'),
])
